src/app/components/client/ExposureCard.tsx

```diff
@@ -42,77 +42,81 @@ export const ExposureCard = (props: Props) => {
     exposureType,
     exposureDetailsLink,
     dateFound,
-    statusPillContent,
     statusPillType,
   } = props;
 
   const [detailsOpen, setDetailsOpen] = useState(false);
+  const [isMobile, setIsMobile] = useState(false);
+
+  useEffect(() => {
+    const handleResize = () => {
+      const width = window.innerWidth;
+      setIsMobile(width < 768); // $screen-md
+    };
+
+    window.addEventListener('resize', handleResize);
+    handleResize();
+
+    return () => window.removeEventListener('resize', handleResize);
+  }, []);
+
+
+  const date = new Date(dateFound * 1000); // Multiply by 1000 to convert seconds to milliseconds
+
+  const formattedDate = date.toLocaleDateString("en-US", {
```

Comment:
Q: should this do some user locale detection instead of forcing en-US locale formatting?

Reply:
If we ever expose this, by the time it happens we'll forget about this hard-coded locale, so I support Peter's suggestion.